pilot/pkg/config/kube/crd/controller/controller.go

```diff
@@ -206,23 +206,29 @@ func (c *controller) RegisterEventHandler(typ string, f func(model.Config, model
 		return
 	}
 	c.kinds[typ].handler.Append(func(old, curr interface{}, ev model.Event) error {
-		curritem, ok := curr.(crd.IstioObject)
-		if ok {
-			config, err := crd.ConvertObject(s, curritem, c.client.domainSuffix)
-			if err != nil {
-				log.Warnf("error translating object for schema %#v : %v\n Object:\n%#v", s, err, curr)
-			} else {
-				olditem, ok := old.(crd.IstioObject)
-				oldconfig := &model.Config{}
-				if ok {
-					oldconfig, err = crd.ConvertObject(s, olditem, c.client.domainSuffix)
-					if err != nil {
-						log.Warnf("error translating object for schema %#v : %v\n Object:\n%#v", s, err, old)
-					}
-				}
-				f(*oldconfig, *config, ev)
-			}
+		var curritem, olditem crd.IstioObject
+		var currconfig, oldconfig *model.Config
```

Comment:
nit: var names should be in camel format

Reply:
Thanks. Fixed.PTAL